### PR TITLE
Update boilerplate to Buidler plugin 0.1.0

### PR DESCRIPTION
**Pull request by eternauta1337**
Implements new hooks interface and simplifies Buidler config file.

### CODE REVIEW

File: buidler.config.js [hunk 1]
```diff
@@ -6,24 +6,8 @@ module.exports = {
...
-        },
-        {
-          privateKey: '0xce8e3bda3b44269c147747a373646393b1504bfcbb73fc9564f5d753d8116608',
-          balance: '100000000000000000000'
-        }
-      ]
-    }
   },
   solc: {
     version: '0.4.24'
```
Comment (kanaagawa) [inline buidler.config.js]: Do we want to have a pre configured amount of runs here?

File: package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "@aragon/abis": "^1.1.0"
   },
   "devDependencies": {
-    "@aragon/buidler-aragon": "^0.0.7",
+    "@aragon/buidler-aragon": "^0.1.0-beta.2",
```
Comment (kanaagawa) [inline package.json]: Reminder to go out of beta here.

File: scripts/buidler-hooks.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+/*
```
Comment (kanaagawa) [inline scripts/buidler-hooks.js:1]: Amazing 🤩

**Review by 0xGabi — APPROVED** [review]
Looks great. I wonder at what point we want to think on deprecate the react boilerplate.

File: buidler.config.js (hunk 1 above)
Comment (eternauta1337) [reply buidler.config.js]: Hmmm. Yeah, I'll add that in.

**Pull request merged**